packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -155,7 +155,7 @@ export class AtomStepsModal {
           onAtomPrimaryClick={this.handlePrimaryClick}
           onAtomSecondaryClick={this.handleSecondaryClick}
           isOpen={this.isOpen}
-          hasFooter={false}
+          hasFooter={true}
```

Comment:
question: what is the default? could it create problems in places that already use it with `hasFooter={false}`?

Reply:
The default hasFooter is `true`, when it is set to `false` it should not show the buttons, for some reason it was marked as false in this component. Could it have been something related to this [PR](https://github.com/juntossomosmais/atomium/pull/713/files#diff-63740e09a4ef6327d67677581cd2b841b8ee853bfe5909551a4f63ecc84d761e)? 🤔